demo/build.gradle

```diff
@@ -16,7 +16,10 @@ import org.apache.tools.ant.filters.ConcatFilter
  * limitations under the License.
  */
 
-apply plugin: 'com.android.application'
+plugins {
+    id 'com.android.application'
+    id 'com.google.secrets_gradle_plugin' version '0.4'
```

Comment:
IIRC the coordinates and version number have changed since this initial PR?

Reply:
Yep—will update this PR with the new coordinates and version